jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -604,6 +604,10 @@ private String appendProxyUserToURL(String url, String user, String propertyKey)
 
   // only add tags for hive jdbc
   private String appendTagsToURL(String url, InterpreterContext context) {
+    if (!Boolean.parseBoolean(getProperty("zeppelin.jdbc.hive.engines.tag.enable"))) {
```

Comment:
set default value as `true` in case `zeppelin.jdbc.hive.engines.tag.enable` is not set when user upgrade zeppelin from old versions

Reply:
> set default value as `true` in case `zeppelin.jdbc.hive.engines.tag.enable` is not set when user upgrade zeppelin from old versions

Good point!